money-monkey-web/app/ui/NewExpenseModal.tsx

```
@@ -37,59 +37,75 @@ export default function NewExpenseModal({
             >
               <div className="relative text-center w-full h-10">
                 <h2 className="p-2 font-bold mb-2 underline">New Expense</h2>
                 <button
-                  className="absolute top-0 right-[-0.5rem] hover:bg-gray-400 rounded-full p-2 m-0"
-                  onClick={() => setVisible(false)}
+                  className="absolute top-0 right-[-0.5rem] rounded-full hover:bg-gray-400 p-0 hover:p-2 m-2 hover:m-0 transition-all "
+                  onClick={(e) => {
+                    setVisible(false);
+                    props.handleReset(e);
+                  }}
                 >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)